multi-config/Chart.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: multi-config-vector
+version: 1.0.0
+description: A demonstration of multiple ConfigMaps being merged into a single Vector config.
+dependencies:
+  - name: vector-aggregator
+    version: ~0.19.1
```

As noted offline this should be changed to the new chart: https://github.com/vectordotdev/helm-charts/blob/develop/charts/vector/README.md

```suggestion
  - name: vector
    version: ~0.2.3
```